bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
It is just my opinion, I don't say they should not live together but the file and package name could be renamed to reflect their new goal and purpose. If all others do not agree on that, don't bother :-D

PS: note that travis is failing :-)